schemas/2016-03-30/Microsoft.DataCatalog.json

```diff
@@ -0,0 +1,116 @@
+                },
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                }
+              ],
+              "description": "Microsoft.DataCatalog/catalogs: units - specifies the number of users, in groups of 100, permitted in the user list."
+            },
+            "enableAutomaticUnitAdjustment": {
+              "oneOf": [
+                {
+                  "type": "boolean"
+                },
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                }
+              ],
+              "description": "Microsoft.DataCatalog/catalogs: enableAutomaticUnitAdjustment - specifies whether to permit automatic unit adjustment. This property is required to add security groups to the user list."
+            },
+            "sku": {
+              "oneOf": [
+                {
+                  "enum": [
+                    "Free",
+                    "Standard"
+                  ]
+                },
+                { "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression" }
+              ],
+              "description": "Microsoft.DataCatalog/catalogs: sku - specifies desired SKU for the catalog (Free, Standard)"
+            },
+            "admins": {
+              "oneOf": [
+                {
+                  "type": "array",
+                  "items": {
+                    "oneOf": [
+                      { "$ref": "#/definitions/user" },
+                      { "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression" }
+                    ]
+                  }
+                },
+                { "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression" }
+              ],
+              "description": "Microsoft.DataCatalog/catalogs: admins - specifies the list of catalog administrators"
+            },
+            "users": {
+              "oneOf": [
+                {
+                  "type": "array",
+                  "items": {
+                    "oneOf": [
+                      { "$ref": "#/definitions/user" },
+                      { "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression" }
+                    ]
+                  }
+                },
+                { "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression" }
+              ],
+              "description": "Microsoft.DataCatalog/catalogs: users - specifies the list of catalog users"
+            }
+          },
+          "required": [
+            "sku"
+          ]
+        }
+      },
+      "required": [
+        "type",
```

Comment:
double check, is "location" required for catalogs? if so, please add it into "required" array (not need to define it in properties though)